src/RMPI/onesided.jl

```diff
@@ -113,9 +113,9 @@ Deposit a vector `buf` into the MPI window `s` on rank `targetrank`. If
     @boundscheck len ≤ length(buf) && len ≤ s.capacity ||
         error("Not enough space left in buffer")
     b_buffer = MPI.Buffer(buf, len, s.DT_b)
-    MPI.Put(b_buffer, targetrank, s.b_win)
+    MPI.Put!(b_buffer, targetrank, s.b_win)
     l_buffer = MPI.Buffer([len,], 1, s.DT_l)
-    MPI.Put(l_buffer, targetrank, s.l_win)
+    MPI.Put!(l_buffer, targetrank, s.l_win)
```

Comment:
Hmm, is this the change that the `MPI.jl` folks promised to do ages ago but sat on for a long time? But then the new code won't be backward compatible any more. We will need to change the compat entries in `Project.toml`.

As far as I can see the bang appeared in `v20.0`.

Reply:
Done.